lwotg/lwotg.go

```diff
@@ -182,6 +183,17 @@ func (s *Server) SetConfig(ctx context.Context, req *otg.SetConfigRequest) (*otg
 	return &otg.SetConfigResponse{}, nil
 }
 
+// GetConfig retrieves the current OTG configuration from the lwotg instance, implementing the GetConfig
+// RPC.
+func (s *Server) GetConfig(_ context.Context, req *emptypb.Empty) (*otg.GetConfigResponse, error) {
```

Comment:
Subjective nit: I would either entirely replace the name "req" or at least prefix it with an underscore to signal that it's unused, similar to how it's done with the context arg

Reply:
makes sense to me -- updated.